tests/SideBySide/DataTypes.cs

```diff
@@ -549,6 +549,49 @@ public void QueryZeroDateTime(bool convertZeroDateTime)
 			}
 		}
 
+		[Theory]
+		[InlineData(DateTimeKind.Unspecified, DateTimeKind.Unspecified, true)]
+		[InlineData(DateTimeKind.Unspecified, DateTimeKind.Local, true)]
+		[InlineData(DateTimeKind.Unspecified, DateTimeKind.Utc, true)]
+		[InlineData(DateTimeKind.Utc, DateTimeKind.Unspecified, true)]
+		[InlineData(DateTimeKind.Utc, DateTimeKind.Local, false)]
+		[InlineData(DateTimeKind.Utc, DateTimeKind.Utc, true)]
+		[InlineData(DateTimeKind.Local, DateTimeKind.Unspecified, true)]
+		[InlineData(DateTimeKind.Local, DateTimeKind.Local, true)]
+		[InlineData(DateTimeKind.Local, DateTimeKind.Utc, false)]
+		public void QueryDateTimeKind(DateTimeKind kindOption, DateTimeKind kindIn, bool success)
+		{
+			var csb = AppConfig.CreateConnectionStringBuilder();
+			csb.DateTimeKind = kindOption;
```

Comment:
This line won't compile with Connector/NET so this whole test needs to be `#if !BASELINE`.